sdk/attestation/test-resources.json

```diff
@@ -0,0 +1,70 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "baseName": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().name]",
+      "metadata": {
+        "description": "The base resource name."
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "The location of the resource. By default, this is the same as the resource group."
+      }
+    },
+    "isolatedSigningCertificate": {
+      "type": "string",
+      "defaultValue": ""
+    },
+    "locationShortName": {
+      "type": "string",
+      "defaultValue": ""
+    }
+  },
+  "variables": {
+    "isolatedTenantName": "[concat(parameters('baseName'), 'iso')]",
```

Comment:
We had talked about adding `concat('js'(`...`)` to this to create JS specific instances (otherwise JS, .Net and Python will collide).

Reply:
oops, fixed.